torch/testing/_internal/dynamo_test_failures.py

```diff
@@ -1,4 +1,7 @@
 # mypy: ignore-errors
+import os
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Disclaimer that I'm not very experienced with this

This file is located in torch, so `__file__` refers to where ever the torch pip package is installed.  Locally this usually works because we do an editable install, but in CI, we build the package and install that normally, so its in site-packages or something like that.  If you want it to show up in the pip package I think it needs to be part of the build somehow, but I'm not sure that would be a good idea 

Some solutions might be:
1. generate a file or something similar prior to calling anything from torch (probably in run_tests.py) that can then be read by common_utils  
2. inspect the stack to get the calling file (the actual test file located in test/, like test/test_ops.py, which is not located in torch) and somehow get back to this folder to generate the list

Both of these solutions have their pros and cons and will probably need to be aware of where the base directory is (in CI I think its test/) and will probably require some path string parsing weirdness

Reply:
What about teaching setup.py (or torchgen, etc) to build a file list based on the directory structure? I assume we have some generated files that end up in the pip wheel in a similar fashion. That way the merging is workable but the loaded file is "just python".